### PR TITLE
Added HMIP-DLD Integration

### PR DESCRIPTION
**Pull request by tobim8**
## Proposed change
As per this Thread https://community.home-assistant.io/t/homematic-ip-request-add-new-hmip-dld-door-lock/297388/21 i found a workaround and want to share with others

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: https://community.home-assistant.io/t/homematic-ip-request-add-new-hmip-dld-door-lock/297388/21 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @tobim8, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19151).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/homematic.markdown [hunk 1]
```diff
@@ -440,6 +440,37 @@ action:
...
+
+There is no avaliable default integration for HMIP Doorlock (HMIP-DLD) in the current pyhomeatic implementation.
+A workaround is to define a template lock in configuration.yaml:
+
+```yaml
+lock:
+  - platform: template
+    name: Basedoor
+    unique_id: basedoor
+    value_template: "{{ states.homematic.ccu2.attributes.base_lock_status }}"
```
Comment (ludeeus) [inline source/_integrations/homematic.markdown]: Use `state_attr(...` <https://www.home-assistant.io/docs/configuration/templating/#states>

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: source/_integrations/homematic.markdown (hunk 1 above)
Comment (tobim8) [reply source/_integrations/homematic.markdown]: Adapted state_attr().

Comment (tobim8) [reply source/_integrations/homematic.markdown]: would be happy to get a hint why the checks are now failing. Compared my syntax again with the documentation and i cant find the problem.

Comment (ludeeus) [reply source/_integrations/homematic.markdown]: When using templates you have to wrap the section with `{% raw %}` / `{% endraw %}`

File: source/_integrations/homematic.markdown [hunk 2]
```diff
@@ -440,6 +440,41 @@ action:
...
+```yaml
+lock:
+  - platform: template
+    name: Basedoor
+    unique_id: basedoor
+    value_template: "{{ state_attr('homematic.ccu2', 'base_lock_status') }}"
+    lock:
+      service: homematic.set_device_value
+      data:
+        address: 002A1BE9A792D2
```
Comment (ludeeus) [inline source/_integrations/homematic.markdown]:
```suggestion
        address: "002A1BE9A792D2"
```

File: source/_integrations/homematic.markdown [hunk 3]
```diff
@@ -440,6 +440,41 @@ action:
...
+        channel: 1
+        param: LOCK_TARGET_LEVEL
+        value: 0
+    unlock:
+      service: homematic.set_device_value
+      data:
+        address: 002A1BE9A792D2
```
Comment (ludeeus) [inline source/_integrations/homematic.markdown]:
```suggestion
        address: "002A1BE9A792D2"
```

File: source/_integrations/homematic.markdown [hunk 4]
```diff
@@ -440,6 +440,41 @@ action:
...
+        channel: 1
+        param: LOCK_TARGET_LEVEL
+        value: 1
+```
+
+{% endraw %}
+
+To get the current value of the current lock status, you have to create a system variable (in example above it is `base_lock_status`) and create a program on CCU, which updates the variable with every change of the Lock level to `true` for locked and `false` for unlocked.
```
Comment (ludeeus) [inline source/_integrations/homematic.markdown]:
```suggestion
To get the current value of the current lock status, you have to create a system variable (in the example above it is `base_lock_status`) and create a program on CCU, which updates the variable with every change of the Lock level to `true` for locked and `false` for unlocked.
```

File: source/_integrations/homematic.markdown [hunk 5]
```diff
@@ -440,6 +440,41 @@ action:
     entity_id: lock.leq1234567
 ```
 
+
+#### Integrating HMIP-DLD
+
+There is no available default integration for HMIP Doorlock (HMIP-DLD) in the current pyhomeatic implementation.
+A workaround is to define a template lock in configuration.yaml:
```
Comment (ludeeus) [inline source/_integrations/homematic.markdown]:
```suggestion
A workaround is to define a template lock in your configuration:
```

File: source/_integrations/homematic.markdown [hunk 6]
```diff
@@ -440,6 +440,41 @@ action:
     entity_id: lock.leq1234567
 ```
 
+
+#### Integrating HMIP-DLD
+
+There is no available default integration for HMIP Doorlock (HMIP-DLD) in the current pyhomeatic implementation.
```
Comment (ludeeus) [inline source/_integrations/homematic.markdown]:
```suggestion
There is no available default integration for HMIP Doorlock (HMIP-DLD) in the current `pyhomeatic` implementation.
```

**Review by ludeeus — APPROVED** [review]

**Pull request merged**